README.md

```diff
@@ -350,6 +351,18 @@ mappings:
 
 Possible values for `match_metric_type` are `gauge`, `counter` and `timer`.
 
+### Time series expiration
+
+`ttl` parameter can be used to define expiration time for stale metrics.
+Value is a time duration with valid time units: "ns", "us" (or "µs"),
+"ms", "s", "m", "h". For example, `ttl: 1m20s`. `0` value is used to indicate
+not expired metrics.
+
+Expiration is useful to gather metrics with changing label values from
```

Comment:
I would rather not cite Kubernetes (pod) as an example here – in that case I would always recommend running the exporter as a side-car in every pod, so that the pod label can be added by Prometheus at scrape time. That also solves a whole bunch of other label consistency and expiry issues, by tying the life cycles of the application and the exporter together.

Can you delete this paragraph please?

Reply:
Paragraph is deleted